snakePipes/workflows/ATAC-seq/internals.snakefile

```diff
@@ -6,5 +6,45 @@ import subprocess
 samples = [ os.path.basename(f) for f in glob.glob('filtered_bam/*.filtered.bam') ]
 samples = [ os.path.basename(x).replace('.filtered.bam','') for x in samples ]
 
-if sampleSheet:
-    cf.check_sample_info_header(sampleSheet)
+if sampleSheet and not os.path.isfile(sampleSheet):
+    print("ERROR: Cannot find sample sheet file! ("+sampleSheet+")\n")
+    exit(1)
+
+if sampleSheet and not cf.check_sample_info_header(sampleSheet):
+    print("ERROR: Please use 'name' and 'condition' as column headers in sample sheet file! ("+sampleSheet+")\n")
+    exit(1)
+
```

Comment:
I moved these two checks above into the cf.check_sample_info_header() function in internal branch,so we just need this to check:
if sampleSheet:
    f sampleSheet and not os.path.isfile(sampleSheet):
        cf.check_sample_info_header(sampleSheet)

and get rid of the two if-s